src/app/home/home.html

```diff
@@ -15,10 +15,10 @@
   </button>
   <button
     md-button
-    [class.highlight]="currentTab === 'commit'"
-    [routerLink]="['/home', { tab: 'commit' }]"
+    [class.highlight]="currentTab === 'ask'"
+    [routerLink]="['/home', { tab: 'ask' }]"
   >
-    COMMIT
+    ASK
```

Comment:
That would be too long for the iPhone 5. Smaller font looks not so nice. @yuttadhammo, what would you like to have?